src/metabase/models/card.clj

```diff
@@ -576,6 +680,11 @@
     (parameter-card/upsert-or-delete-from-parameters! "card" (:id card) (:parameters card))
     (query-analysis/analyze! card)))
 
+(defn- apply-dashboard-question-updates [changes]
+  (if (:dashboard_id changes)
```

Comment:
I think this would read better as an if-let, since you use `(:dashboard_id changes)` on the true branch.